docs/tutorial/CustomDataLoaderOnSL.ipynb

```diff
@@ -76,108 +95,81 @@
-   "metadata": {},
-   "source": [
-    "## 数据集介绍"
-   ]
-  },
-  {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "f144e65c",
    "metadata": {},
    "source": [
-    "我们这里将使用最经典的MovieLens数据集来进行演示。  \n",
-    "MovieLens是一个开放式的推荐系统数据集，包含了电影评分和电影元数据信息。  \n",
-    "[数据集地址](https://secretflow-data.oss-accelerate.aliyuncs.com/datasets/movielens/ml-1m.zip)"
+    "## Datasets"
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "798f117c",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
-    "我们对数据进行了切分：\n",
+    "We will use the classic dataset [MovieLens](https://secretflow-data.oss-accelerate.aliyuncs.com/datasets/movielens/ml-1m.zip) for introduction.MovieLens is an open recommendation system dataset that contains movie ratings and movie metadata information.\n",
+    "\n",
+    "And we split data into several pieces:\n",
     "- alice: \"UserID\", \"Gender\", \"Age\", \"Occupation\", \"Zip-code\"\n",
-    "- bob:   \"MovieID\", \"Rating\", \"Title\", \"Genres\", \"Timestamp\""
+    "- bob: \"MovieID\", \"Rating\", \"Title\", \"Genres\", \"Timestamp\"\n"
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "7a6ef523",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
    "source": [
-    "## 定义DataBuilder函数  "
+    "## Define DataBuilders"
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "abcd3be2",
    "metadata": {},
    "source": [
-    "我们定义DataBuilder的目的是已有的FedDataFrame和FedNdarray提供的功能无法满足需求，我们可以在通过自定义DataBuilder来满足高阶定制化需求。  \n",
-    "在SplitLearning中（各方数据需要对齐）我们暂时仅提供了面向CSV数据的DataBuilder能力。在自定义DataBuilder中可以定义对每一行数据怎么处理。SLModel会根据您定义的方式进行操作。\n",
+    "We customize a DataBuilder with the aim to meet higher level customization demands, as the existing FedDataFrame and FedNdarray did not provide the required functionality.\n",
     "\n",
-    "需要注意的点:  \n",
-    "- 我们的MovieLens数据集读取进来后会是CSV格式的，但是需要被当做稀疏特征来进行处理，所以会在DataBuilder中将每一列转成字典形式\n",
-    "- 在Bob测我们将评分进行了Threshold处理成二值形式\n",
-    "- 在DataBuilder中可以定义自定义函数，然后使用dataset.map来应用到每一列。\n",
-    "- 因为垂直模式需要保证两方数据是对齐的，所以暂时只支持CSV模式\n",
-    "- CSV模式只需要返回Dataset，Dataset需要定义好Batch_size，Repeat，SLModel会根据Dataset推断Steps_per_epoch"
+    "In SplitLearning(where all sides of the data need to be aligned), we only provide DataBuilder for CSV data for the time being. You can define what to do with each row of data in the custom DataBuilder. The operations in SLModel depend on the way you define it.\n",
+    "\n",
+    "The things you should notice here:\n",
+    "- We read MovieLens dataset in CSV format. However, it needs to be treated as sparse features. We should convert each column in the DataBuilder to dictionary form.\n",
+    "- Bob's score was processed into binary form using threshold.\n",
+    "- Custom functions can be defined in the DataBuilder and then applied to each column using dataset.map.\n",
```

Comment:
`We should convert each column in the DataBuilder to dictionary form` --> `The DataBuilder will convert each column into the form of dictionary`